.github/workflows/build.yml

```diff
@@ -165,52 +172,61 @@ jobs:
             sudo apt-get -qq update
             sudo apt-get install libjpeg-dev libpng-dev zlib1g-dev libsdl2-dev
           fi
-          cmake -E make_directory ${{ runner.workspace }}/build
+          cmake -E make_directory ${{ github.workspace }}/build
 
       - name: Configure CMake
         shell: bash
-        working-directory: ${{ runner.workspace }}/build
+        working-directory: ${{ github.workspace }}/build
         run: |
+          OPTIONS="-DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/install"
           if [ "${{ matrix.portable }}" == "Portable" ]; then
-            OPTIONS="-DUseInternalLibs=ON -DBuildPortableVersion=ON -DCMAKE_INSTALL_PREFIX=${{ runner.workspace }}/install"
+            OPTIONS+=" -DUseInternalLibs=ON -DBuildPortableVersion=ON"
           else
-            OPTIONS="-DUseInternalLibs=OFF -DBuildPortableVersion=OFF -DCMAKE_INSTALL_PREFIX=${{ runner.workspace }}/install"
+            OPTIONS+=" -DUseInternalLibs=OFF -DBuildPortableVersion=OFF"
           fi
-          
           if [ ${{ matrix.arch }} == "x86" ]; then
-          cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} $OPTIONS -DCMAKE_TOOLCHAIN_FILE=cmake/Toolchains/linux-i686.cmake
-          else
-          cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} $OPTIONS
+            OPTIONS+=" -DCMAKE_TOOLCHAIN_FILE=cmake/Toolchains/linux-i686.cmake"
           fi
+          OPTIONS+=" -DBuildJK2SPEngine=ON -DBuildJK2SPGame=ON -DBuildJK2SPRdVanilla=ON"
+          cmake $GITHUB_WORKSPACE $OPTIONS
 
       - name: Build
-        working-directory: ${{ runner.workspace }}/build
+        working-directory: ${{ github.workspace }}/build
         shell: bash
         run: cmake --build .
 
       - name: Install
-        if: ${{ matrix.build_type == 'Release' }} 
-        working-directory: ${{ runner.workspace }}/build
+        if: ${{ matrix.build_type == 'Release' }}
+        working-directory: ${{ github.workspace }}/build
         shell: bash
         run: cmake --install .
 
-      - name: Create binary archive
-        if: ${{ matrix.build_type  == 'Release'}}
-        working-directory: ${{ runner.workspace }}/install/JediAcademy
+      - name: Create OpenJK binary archive
+        if: ${{ matrix.build_type == 'Release'}}
+        working-directory: ${{ github.workspace }}/install/JediAcademy
         shell: bash
         run: |
-          if [ ${{ matrix.arch }} == "x86" ]; then
```

Comment:
The `chmod +x` doesn't appear to be necessary. It also doesn't apply to the SP binary.
We can reintroduce it if necessary